tools/runners/test262-harness.py

```diff
@@ -585,21 +580,21 @@ def get_parameter(match):
 
     @staticmethod
     def execute(command):
-        if is_windows():
-            args = '%s' % command
-        else:
-            args = command.split(" ")
+        args = command.split(" ")
```

Comment:
We still need this trick, because run-test-suite-test262.py concatenates args.runtime and args.engine with space.

Reply:
> We still need this trick, because run-test-suite-test262.py concatenates args.runtime and args.engine with space.

Do you means es5.1 need this trick?

Reply:
But according to https://github.com/jerryscript-project/jerryscript/pull/4378/checks?check_run_id=1620216059
 run-test-suite-test262.py are not affected

Reply:
@ossy-szeged Please show me the full command that need to running these tests.

Reply:
As you wish, revert it first, there is no need do this here @ossy-szeged

Reply:
> @ossy-szeged Please show me the full command that need to running these tests.

tools/runners/run-test-suite-test262.py --runtime qemu-***
(or you can use RUNTIME environment variable too instead of command line option)